### ISSUE TITLE
Why `$$` math blocks are parsed as `invalid`?

### ISSUE THREAD

**Issue by byte-sourcerer** [mava/Nova-LaTeX#7]
https://github.com/mava/Nova-LaTeX/blob/8e0009865df917cb31780e1f098834d2b72ed273/LaTeX.novaextension/Syntaxes/LaTeX.xml#L46

The following math block is parsed as `latex.math.displaystyle.invalid`, which results in strange highlighting...

```
$$
\bar{\beta}_i(t) = 
\begin{cases}
	0 & t \ge \tau_i, \\
	1 & t < \tau_i
\end{cases}
$$
```
<img width="821" alt="CleanShot-2022-06-08-16 04 21" src="https://user-images.githubusercontent.com/23581749/172564920-2b4cfd05-1ca1-4a62-96a4-367ebeea1954.png">

**Comment by mava** [comment]
Because [“LaTeX doesn’t officially support $$.”](https://tex.stackexchange.com/a/69854) See also [this](https://tex.stackexchange.com/q/40492) and [this.](https://www.texdev.net/2022/09/13/math-mode-%24-%24-vs-%5C(-%5C))

**Comment by mava** [comment]
Fixed in [v0.8](https://github.com/mava/Nova-LaTeX/releases/tag/v0.8).
